.github/workflows/include_latest-version.yml

```diff
@@ -0,0 +1,43 @@
+name: Load Supported Versions
```

Comment:
As you stated in the name, it might be better to keep the file name plural as latest version is exposed for multi tools, what do you think ?

That would be: .github/workflows/include_latest-versions.yml